documentation/migrations/0026_auto_20200616_0122.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('documentation', '0025_auto_20200616_0118'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='example',
+            name='embed_app_with_code_height',
```

Comment:
can you please consolidate these into a single migration? it looks like all 3 affect the same field.

Reply:
Sounds good, will do